include/flatbuffers/util.h

```diff
@@ -636,6 +636,34 @@ inline bool EscapeString(const char *s, size_t length, std::string *_text,
   return true;
 }
 
+inline std::string EscapeAndWrapBuffer(const void *buf, size_t bufsize,  size_t max_length,
+                                       const std::string wrapped_line_prefix,
+                                       const std::string wrapped_line_suffix) {
+    std::string text= wrapped_line_prefix;
```

Comment:
Use `auto` where possible.
Use formatting like the rest of the code (see e.g. `=` with missing spacing), or run `git clang-format`.
No unnecessary blank lines after `}` etc.